src/main/java/uk/gov/hmcts/reform/iacasedocumentsapi/domain/service/WordDocumentToPdfConverter.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.hmcts.reform.iacasedocumentsapi.domain.service;
+
+import static java.io.File.createTempFile;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.StandardCopyOption;
+import org.apache.commons.io.IOUtils;
+import org.springframework.core.io.Resource;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.iacasedocumentsapi.domain.FileType;
+import uk.gov.hmcts.reform.iacasedocumentsapi.infrastructure.clients.DocmosisDocumentConversionClient;
+
+@Service
+public class WordDocumentToPdfConverter {
```

Comment:
Think this is fine, but maybe could be some kind of Converter interface in future.